src/ADAL.PCL.iOS/WebUI.cs

```diff
@@ -49,7 +49,16 @@ public WebUI(IPlatformParameters parameters)
         public async Task<AuthorizationResult> AcquireAuthorizationAsync(Uri authorizationUri, Uri redirectUri, CallState callState)
         {
             returnedUriReady = new SemaphoreSlim(0);
+
+            string requestId = Telemetry.GetInstance().CreateRequestId();
+            Telemetry.GetInstance().StartEvent(requestId, "ui_event");
+
+            UIEvent UiEvent = new UIEvent();
+
             Authenticate(authorizationUri, redirectUri, callState);
+
+            Telemetry.GetInstance().StopEvent(requestId, UiEvent, "ui_event");
```

Comment:
>Telemetry [](start = 12, length = 9)

I gave you an in person feedback and advice to update MockWebUI to include "wait times". That way you will be able to validate that ui event is actually working. You agreed to implement it and I still don't see that as a part of the code review. The advice I parted has been largely ignored and we still have missing test coverage and bugs. :(